modular_zubbers/code/modules/languages/panslavic.dm

```diff
@@ -1,3 +1,23 @@
-/datum/language/panslavic
+/datum/language/spinwarder
```

Comment:
Is it intentional to change the typepath and not the name? Usually you want typepaths to match the name or close to it